pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -458,6 +458,41 @@ type EndpointIdentity struct {
 	LabelsSHA256 string `json:"labelsSHA256,omitempty"`
 }
 
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CiliumIdentity is an identity managed by Cilium
```

Comment:
Nit: this documentation is a bit too terse to me - perhaps refer to the identity structure in `pkg/identity`? Or even better, have some of what you put in the commit message here, since that describes its lifecycle / usage in a given Cilium deployment vs. just what it "is".